File: pgm/ABC_Java/ABC_195_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final String Yes = "Yes";
        final String No = "No";
        final int one = 1;
        final int zero = 0;
        final int M = Integer.parseInt (arr[zero]);
        final int H = Integer.parseInt (arr[one]);
        System.out.println (H % M == zero ? Yes : No);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_195_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.math.BigDecimal
// import java.math.RoundingMode
// import java.time.LocalTime
// import java.time.format.DateTimeFormatter
// import java.util.Scanner
// import java.util.StringTokenizer
// import java.util.TreeMap

// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val space: String = " "
const val zero: Int = 0

fun main() {
    val (M: Int, H: Int) = readln().split(space).map { it.toInt() }
    if (H % M == zero) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}